### PR TITLE
[CI:DOCS] socket_activation.md: Add start/stop sections

### PR DESCRIPTION
**Pull request by eriksjolund**
* Add section "Starting a socket-activated service".

* Add section "Stopping a socket-activated service".

* Clarify in the diagrams that socket activation
  only happens for the first client connection.

Signed-off-by: Erik Sjölund <erik.sjolund@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: docs/tutorials/socket_activation.md [hunk 1]
```diff
@@ -55,6 +55,9 @@ $ export DOCKER_HOST=unix://$XDG_RUNTIME_DIR/podman/podman.sock
 $ docker-compose up
 ```
 
+When __docker-compose__ connects to the UNIX socket `$XDG_RUNTIME_DIR/podman/podman.sock`,
```
Comment (vrothberg) [inline docs/tutorials/socket_activation.md:58]:
```suggestion
When __docker-compose__ or any other client connects to the UNIX socket `$XDG_RUNTIME_DIR/podman/podman.sock`,
```

Comment (vrothberg) [reply docs/tutorials/socket_activation.md:58]: Just to avoid confusion if readers use another client.

**Review by vrothberg — COMMENTED** [review]
Just a tiny nit but LGTM

Thanks so much for you contributions, @eriksjolund

**Review by vrothberg — COMMENTED** [review]
LGTM
@containers/podman-maintainers PTAL

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**